resources/views/checkout.blade.php

```diff
@@ -23,9 +70,10 @@
 <!-- Navbar -->
 <nav class="navbar navbar-expand-sm navbar-transparent navbar-dark navbar-theme-primary mb-4">
     <div class="container position-relative">
-        <a class="navbar-brand me-lg-5" href="{{ route('main-site') }}">
-            <img class="navbar-brand-dark" src="{{ asset('images/brand/light.svg') }}" alt="Volt logo"/>
-            <img class="navbar-brand-light" src="{{ asset('images/brand/dark.svg') }}" alt="Volt logo"/>
+        <a class="navbar-brand me-lg-5" href="{{ route('home') }}">
+            @if (file_exists(storage_path("app/public/images/".$settings->custom_icon_filename)) && $settings->custom_icon_filename != null)
```

Comment:
`$settings->custom_icon_filename != null` should be `!is_null($settings->custom_icon_filename)`